package.json

```diff
@@ -2454,6 +2463,16 @@
                     }
                 }
             },
+            {
+                "command": "aws.copyLambdaUrl",
+                "title": "%AWS.generic.copyUrl%",
```

Comment:
My rationale for this was that 'Copy URL' is a generic command. API gateway defines its own title for 'Copy URL' as well under `AWS.command.apig.copyUrl`. So eventually as we add more copy url commands they can all use the same one since it is just the display name in the context menu.